cmd/health_ready.go

```diff
@@ -0,0 +1,26 @@
+package cmd
```

Comment:
Please apply the comments from `health_alive` here as well!

Reply:
Done